next/content/docs/guides/nextjs.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Connect a Next.js application to Neon
+enableTableOfContents: true
+redirectFrom:
+  - /docs/quickstart/vercel
+  - /docs/integrations/vercel
+---
+
+Next.js by Vercel is an open-source web development framework that enables React-based web applications. This topic describes how to create a Neon project and access it from a Next.js application.
+
+To create a Neon project and access it from a Next.js application:
+
+1. [Create a Neon project](#create-a-neon-project)
+2. [Create a Next.js project](#create-a-nextjs-project)
+3. [Add a PostgreSQL client to your app](#add-a-postgresql-client-to-your-app)
+4. [Add your Neon connection details](#add-your-neon-connection-details)
+5. [Connect to the Neon database](#connect-to-the-neon-database)
+
+## Create a Neon project
+
+If you do not have one already, create a Neon project. Save your connection details including your password. They are required when defining connection settings.
+
+1. Navigate to the [Projects](https://console.neon.tech/app/projects) page in the Neon Console.
+2. Click **New Project**.
+3. Specify a name, a PostgreSQL version, a region, and click **Create Project**.
+
+For additional information about creating projects, see [Set up a project](/docs/get-started-with-neon/setting-up-a-project).
+
+## Create a Next.js project
+
+Create a Next.js project if you do not have one. For instructions, see [Create a Next.js App](https://nextjs.org/learn/basics/create-nextjs-app/setup), in the Vercel documentation.
+
+## Add a PostgreSQL client to your app
+
+Add a PostgreSQL client to your app, such as `Postgres.js`. For instructions, refer to the [postgres.js Getting started](https://www.npmjs.com/package/postgres).
```

Comment:
Comment from Anna in the old PR: is there a template available in Vercel that uses Postgres.js already? maybe we can refer to that as an example?

Reply:
Nope, there are none. It could be cool to cross-reference our serverless driver though

Reply:
Perhaps we can offer some guidance instead, explaining what the workflow is like: Add the integration, make sure the variables in your app match those in Vercel, make a change in GitHub, redeploy the app to see the preview branch action in Neon?